packages/browser/src/transports/base.ts

```diff
@@ -54,6 +63,48 @@ export abstract class BaseTransport implements Transport {
     return this._buffer.drain(timeout);
   }
 
+  /**
+   * @inheritDoc
+   */
+  public recordLostEvent(type: Outcome): void {
+    logger.log(`Adding ${type} outcome`);
+    this._outcomes[type] = (this._outcomes[type] ?? 0) + 1;
+  }
+
+  /**
+   * Send outcomes as an envelope
+   */
+  protected _flushOutcomes(): void {
+    if (!navigator || typeof navigator.sendBeacon !== 'function') {
+      logger.warn('Beacon API not available, skipping sending outcomes.');
```

Comment:
There's a `fetch` `keepalive` that is a replacement for `sendBeacon`, see https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters